src/test/java/uk/gov/companieshouse/filinghistory/api/controller/FilingHistoryControllerIT.java

```diff
@@ -486,6 +518,43 @@ void shouldReturn503ServiceUnavailableWhenChsKafkaApiReturnsA503ResponseAndDocum
         verify(instantSupplier, times(2)).get();
         WireMock.verify(
                 requestMadeFor(new ResourceChangedRequestMatcher(RESOURCE_CHANGED_URI, getExpectedChangedResource())));
+
+    }
+
+    @Test
```

Comment:
Would be good to add a test case for when the item to delete is not in the database, 404 not found.

Reply:
done